app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/CreateDLCOfferDialog.scala

```diff
@@ -412,7 +412,7 @@ class CreateDLCOfferDialog extends Logging {
                     }
                   } else {
                     // add empty rounding intervals
-                    addRoundingRow(None, None)
+                    addRoundingRow(Some(0), Some(Satoshis(1000)))
```

Comment:
so i don't know much about this to be honest, but when we do add this we definitely want to make this configurable. Maybe a new configuration key `bitcoin-s.dlc`

Reply:
The GUI doesn't have access to the config